src/dashboard/routes/Dashboard.ts

```diff
@@ -3,10 +3,12 @@ import { lazy } from "react";
 import { generatePath } from "react-router-dom";
 import type { RouteDef } from "../../route-config";
 
-export type DashboardParams = { realm?: string };
+export type DashboardTabs = "info" | "providers";
```

Comment:
This type should probably have a singular name like the others.
```suggestion
export type DashboardTab = "info" | "providers";
```